server/src/main/kotlin/suwayomi/tachidesk/manga/impl/download/DownloadManager.kt

```diff
@@ -107,11 +83,11 @@ object DownloadManager {
         }
 
         if (downloader == null) {
-            downloader = Downloader(downloadQueue) { notifyAllClients() }
+            downloader = Downloader(downloadQueue) { eventDispatcher.notifyAllClients(getStatus()) }
             downloader!!.start()
         }
 
-        notifyAllClients()
+        eventDispatcher.notifyAllClients(getStatus())
```

Comment:
Could `notifyAllClients()` maybe have been left in and only its implementation changed to use `eventDispatcher`? It seems like this line is repeated a lot.